tests/test_connection.py

```diff
@@ -269,6 +271,17 @@ def test_connect_in_mocking(self):
         conn = get_connection("testdb7")
         self.assertIsInstance(conn, mongomock.MongoClient)
 
+        disconnect_all()
+
+        class SomeDocument(Document):
```

Comment:
for a newcomer it may sound odd to have to save a Document to actually check the connection's database name, could  you add a quick comment like `#database won't exist until we save a document`